saleor/tests/fixtures.py

```diff
@@ -795,6 +794,66 @@ def color_attribute(db):
     return attribute
 
 
+@pytest.fixture
+def date_attribute(db):
+    attribute = Attribute.objects.create(
+        slug="release-date",
+        name="Release date",
+        type=AttributeType.PRODUCT_TYPE,
+        input_type=AttributeInputType.DATE,
+        filterable_in_storefront=True,
+        filterable_in_dashboard=True,
+        available_in_grid=True,
+    )
+
+    value = datetime.datetime(2020, 10, 5, tzinfo=pytz.utc)
+    AttributeValue.objects.create(
```

Comment:
Can you create them in bulk? The same for `date_time_attribute`. 
Both of them are fixture so would be great to have DB hits as low as possible